package.json

```diff
@@ -19,6 +19,7 @@
     "lint:eslint": "eslint . --cache --ext js,ts",
     "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write",
     "lint:misc": "prettier '**/*.json' '**/*.md' '!CHANGELOG.md' '**/*.yml' '!.yarnrc.yml' --ignore-path .gitignore --no-error-on-unmatched-pattern",
+    "lint:dependencies": "depcheck --config=.depcheckrc.json",
```

Comment:
Nit: It doesn't appear that this `--config` option is required.

Reply:
Ah. I was having some issues when trying to use `.depcheckrc` (without extension) at first, so I tried it with the `--config` option. But you're right that it's not required now.